templates/LDA_Preprocessing.ipynb

```diff
@@ -1,15 +1,941 @@
 {
```

Comment:
<p>I think this information would be better placed in the docstring. Putting it here means it will need to be updated in two places, increasing the chance of errors. You can link to the readthedocs section that has the relevant information instead. Same for the description a few cells above </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angelolab/ark-analysis/pull/468/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>

Reply:
<p>Most of it is already in the docstring, so I can just remove it if you like.</p>

Reply:
Got it, in that case I think linking to the docstring is best

Reply:
Sounds good